login-workflow/src/screens/LoginScreen/types.ts

```diff
@@ -1,8 +1,8 @@
 import { TextFieldProps } from '@mui/material';
-import { WorkflowCardBaseProps } from '../../components/WorkflowCard/WorkflowCard.types';
+import { WorkflowCardProps } from '../../components/WorkflowCard/WorkflowCard.types';
 import { ErrorManagerProps } from '../../components/Error';
 
-export type LoginScreenProps = WorkflowCardBaseProps & {
+export type LoginScreenProps = WorkflowCardProps & {
```

Comment:
We don't really want the LoginScreen to extend everything from WorkflowCardProps (i.e., we do not want it to support header, instructions, or actions). Probably better just to add an explicit login prop on the LoginScreenBase and pass it through to the underlying WorkflowCard without all the extra props we don't intend to support.